packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,352 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
+    <VbCard title="Default">
+      <va-tree-view
+        :nodes="nodes"
+        node-key="id"
+      />
+    </VbCard>
+    <VbCard title="Icons">
+      <va-tree-view
+        :nodes="nodesWithIcons"
+        node-key="id"
+        expand-all
+      >
+        <template #icon="{icon}">
+          <va-icon
+            v-if="icon"
+            :name="icon"
+            size="20px"
+          />
+        </template>
+      </va-tree-view>
+    </VbCard>
+    <VbCard title="Selection and icons">
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <br />
+      <p>
+        {{ JSON.stringify(selectedNodes) }}
+      </p>
+      <br />
+      <va-tree-view
+        v-model="selectedNodes"
+        :nodes="nodesWithIcons"
+        node-key="id"
+        expand-all
+        selectable
+        :selection-type="selectionType"
```

Comment:
Both tree-views use the same v-model. I would expect structure to look the same, yet it doesn't.

<img width="297" alt="image" src="https://user-images.githubusercontent.com/5394573/183222951-f91bda77-0906-4221-a5e4-8c3a44bf0a82.png">

Reply:
It's a single tree view. :)